curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/finding-a-remainder-in-javascript.md

```diff
@@ -13,12 +13,20 @@ The <dfn>remainder</dfn> operator `%` gives the remainder of the division of two
 
 **Example**
 
-<blockquote>5 % 2 = 1 because<br>Math.floor(5 / 2) = 2 (Quotient)<br>2 * 2 = 4<br>5 - 4 = 1 (Remainder)</blockquote>
+```js
+const five = 5;
+const two = 2;
+const quotient = Math.floor(5 / 2);
+console.log(quotient); // logs 2
+```
 
 **Usage**  
 In mathematics, a number can be checked to be even or odd by checking the remainder of the division of the number by `2`.
 
-<blockquote>17 % 2 = 1 (17 is Odd)<br>48 % 2 = 0 (48 is Even)</blockquote>
+```js
+const oddNumber = 17;
+const evenNumber = 48;
+```
```

Comment:
![image](https://user-images.githubusercontent.com/14315133/219605393-e60c3236-cfd2-4a5a-a87b-d0734cd169e9.png)
The issue is still present but I don't have any better ideas I agree writing like javascript may be too confusing for beginners. 
I guess one way to address the issue is to dynamically add styles to address the issues when the course will be available in RTL languages.
If you are ok with it, I'll commit the suggestions.

Reply:
Could you check how that would look in the curriculum? I know there are added some specific styles for the RTL.

Reply:
![image](https://user-images.githubusercontent.com/14315133/219635459-9f61481c-478c-46f1-a4fd-9f13e3047b87.png)
I don't think the curriculum is available in RTL languages yet, only the publications. I don't know how to check in my local copy but I found an article in Arabic that uses the `<pre>` tag and it looks like there is some specific style for RTL.

Reply:
Here are details how it can be tested locally, Arabic can be set via `.env` file https://contribute.freecodecamp.org/#/how-to-work-on-localized-client-webapp?id=testing-the-client-app-in-a-world-language

Reply:
![image](https://user-images.githubusercontent.com/14315133/219869366-78f1a227-cb4f-46f4-a2cf-bcdc904e973b.png)
I noticed there are dedicated files for the arabic languages, I don't understand the needs to fix the RTL layout for the english file.

Reply:
@Sboonny any suggestions? Is it okay to use the `pre` here for now?

Reply:
I have checked the markdown files, I need to add `dir='ltr'` to the `pre` elements anyway, go for it 👍.

Reply:
should I resolve the conversation and commit the suggestions?

Reply:
yeab